### PR TITLE
Drop all mentions of Passenger

### PR DESCRIPTION
**Pull request by ehelms**
We have removed Passenger and the docs should reflect that. Note my intent here was not to try to rewrite any particular section but to drop mentions of Passenger where it is found currently.

### CODE REVIEW

File: _includes/manuals/nightly/3.1.2_hardware_requirements.md [hunk 1]
```diff
@@ -13,6 +13,5 @@ For a bare minimum installation with few clients and no Puppet server, the requi
 
 #### Scaling notes
 
-* The default installation when using Passenger will increase the number of running processes based on the number of simultaneous requests, up to a default maximum of six instances. Each instance will typically require 0.5-1GB of memory, depending on the number of configured plugins.
```
Comment (ekohl) [inline _includes/manuals/nightly/3.1.2_hardware_requirements.md:16]: We should translate this into what's done for Puma (default 2 workers, each worker up to 16 threads).

File: _includes/manuals/nightly/3.2.2_installer_options.md [hunk 2]
```diff
@@ -289,7 +289,7 @@ kafo-export-params -c /etc/foreman-installer/scenarios.d/foreman.yaml -f html
     <tbody>
       <tr>
         <td style='white-space:nowrap'>--foreman-apache</td>
-        <td>Configure Foreman via Apache. By default via passenger but otherwise as a reverse proxy.</td>
+        <td>Configure Foreman via Apache as a reverse proxy.</td>
```
Comment (ekohl) [inline _includes/manuals/nightly/3.2.2_installer_options.md]: Normally we update the installer options via a script: https://github.com/theforeman/theforeman.org/tree/gh-pages/scripts/installer

File: _includes/manuals/nightly/3.2_foreman_installer.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 
 The Foreman installer is a collection of Puppet modules that installs everything required for a full working Foreman setup.  It uses native OS packaging (e.g. RPM and .deb packages) and adds necessary configuration for the complete installation.
 
-Components include the Foreman web UI, Smart Proxy, Passenger (for the puppet master and Foreman itself), and optionally TFTP, DNS and DHCP servers.  It is configurable and the Puppet modules can be read or run in "no-op" mode to see what changes it will make.
```
Comment (ekohl) [inline _includes/manuals/nightly/3.2_foreman_installer.md:4]: Auch, the Puppet part hasn't been true for a long time.

**Review by ekohl — CHANGES_REQUESTED** [review]
Another thing that has changed is `touch tmp/restart.txt` which is no longer supported. It must be `systemctl restart foreman` now. It's mentioned here:
```console
$ rg restart.txt plugins/ _includes/manuals/nightly/
_includes/manuals/nightly/7.2_debugging.md
11:    touch ~foreman/tmp/restart.txt

_includes/manuals/nightly/5.8.3_secret_token.md
4:touch ~foreman/tmp/restart.txt

plugins/index.md
294:    <li>Restart Foreman with <code>touch ~foreman/tmp/restart.txt</code> or <code>service httpd restart</code></li>
310:    <li>Restart Foreman: <code>touch ~foreman/tmp/restart.txt</code> or <code>service apache2 restart</code></li>
369:* Then restart Foreman with `touch ~foreman/tmp/restart.txt`
```

File: _includes/manuals/nightly/3.2.2_installer_options.md (hunk 2 above)
Comment (ehelms) [reply _includes/manuals/nightly/3.2.2_installer_options.md]: I'll revert this file and let the final update of these handle that.

**Review by ekohl — APPROVED** [review]

**Pull request merged**